CHANGELOG.md

```diff
@@ -1,6 +1,30 @@
 # Changelog for AWS X-Ray SDK for JavaScript
-<!--LATEST=3.3.6-->
+<!--LATEST=3.3.7-->
 <!--ENTRYINSERT-->
+
+## 3.3.7
+View [the latest changes](https://github.com/aws/aws-xray-sdk-node/compare/aws-xray-sdk-node%403.3.6...aws-xray-sdk-node%403.3.7)
+
+* Added node 16 testing [PR #513](https://github.com/aws/aws-xray-sdk-node/pull/513)
+* Added opentelemetry JS links to README [PR #514](https://github.com/aws/aws-xray-sdk-node/pull/514)
```

Comment:
Do not include these types of "internal" PRs in the changelog - we only include changes that impact behavior of SDK